alsongDalsong/ASNetworkKit/ASNetworkKit/Firebase/ASFirebaseManager.swift

```diff
@@ -3,13 +3,13 @@ import ASDecoder
 import ASEncoder
 import Combine
 internal import FirebaseAuth
-internal import FirebaseFirestore
-internal import FirebaseDatabase
+@preconcurrency internal import FirebaseFirestore
+@preconcurrency internal import FirebaseDatabase
 import Foundation
 
-public final class ASFirebaseManager: ASFirebaseAuthProtocol, ASFirebaseDatabaseProtocol {
-    private var databaseRef = Database.database().reference()
-    private var firestoreRef = Firestore.firestore()
+public final class ASFirebaseManager: ASFirebaseAuthProtocol, ASFirebaseDatabaseProtocol, Sendable {
```

Comment:
프로토콜 함수들끼리 연관성이 크게 없다면 extension으로 떼어내면 좋을 것 같네요
```suggestion
extension ASFirebaseManager: ASFirebaseAuthProtocol {
```